numpy/core/arrayprint.py

```diff
@@ -617,8 +617,8 @@ def _extendLine(s, line, word, max_line_len, next_line_prefix):
     return s, line
 
 
-def _formatArray(a, format_function, rank, max_line_len,
-                 next_line_prefix, separator, edge_items, summary_insert):
+def _formatArray(a, format_function, rank, max_line_len, next_line_prefix,
+                 separator, edge_items, summary_insert, legacy):
```

Comment:
Maybe default to `=False`, just in case anyone is relying on these internals? Not that we have to support them, but it's easy to do so.

Reply:
Actually, I want to gut this whole function to fix #9816, so let's leave this as is